plugins/discourse-calendar/test/javascripts/acceptance/category-calendar-navigation-test.js

```diff
@@ -0,0 +1,97 @@
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+
+acceptance(
+  "Discourse Calendar - Category Events Calendar Navigation",
+  function (needs) {
+    needs.user();
+    needs.settings({
+      calendar_enabled: true,
+      discourse_post_event_enabled: true,
+      events_calendar_categories: "1|2",
+    });
+
+    let eventCalls = [];
+
+    async function assertCategory(assert, { name, id, count = 1 }) {
+      eventCalls = [];
```

Comment:
Having a helper ensures no possible flakiness due to event calls not being cleaned